scripts/stac/imagery/item.py

```diff
@@ -27,6 +30,10 @@ def __init__(self, id_: str, file: str) -> None:
                 }
             },
             "stac_extensions": [StacExtensions.file.value],
+            "properties": {
+                "created": file_modified_datetime,
```

Comment:
`file_modified_datetime` corresponds the the modified datetime of [the asset (TIFF)](https://github.com/linz/topo-imagery/blob/e71444fefa83b106249c5e7d359f934bae329ca4/scripts/stac/imagery/create_stac.py#L38). 

Based on [the STAC documentation](https://github.com/radiantearth/stac-spec/blob/master/item-spec/common-metadata.md#date-and-time), If you want to store this information, it should be in the `asset`.
The Item `properties.created` and `properties.updated` should 

> identify the creation and update times **of the metadata**

Reply:
Correct, see Alice's mock-ups with `created` and `updated` in the Item properties and in the Item Assets properties: https://toitutewhenua.atlassian.net/jira/software/c/projects/TDE/boards/768?selectedIssue=TDE-1135